### PR TITLE
Upload download docs saga 1687

### PR DESCRIPTION
**Pull request by sahaSonali**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/app/case/CosApiClient.ts [hunk 1]
```diff
@@ -88,22 +88,24 @@ export class CosApiClient {
...
 
-  public async updateCase(user: UserDetails, caseId: string, data: Partial<CaseData>): Promise<CaseWithId> {
-    data.applicantCaseName = 'Tom Jerry - updated';
+  public async updateCase(
+    user: UserDetails,
+    caseId: string,
+    data: Partial<CaseData>,
+    eventId: string
+  ): Promise<CaseWithId> {
+    //data.applicantCaseName = 'Tom Jerry - updated';
```
Comment (AlokDatta) [inline src/main/app/case/CosApiClient.ts:97]: remove this line please

File: src/main/app/case/CosApiClient.ts [hunk 2]
```diff
@@ -88,22 +88,24 @@ export class CosApiClient {
...
     try {
       const headers = {
         Accept: 'application/json',
         'Content-Type': 'application/json',
         Authorization: 'Bearer ' + user.accessToken,
         serviceAuthorization: getServiceAuthToken(),
+        accessCode: 'abcdef',
```
Comment (AlokDatta) [inline src/main/app/case/CosApiClient.ts:104]: line no 104, why?

File: src/main/app/document/DocumentManagementController.test.ts [hunk 3]
```diff
@@ -82,9 +82,9 @@ describe('DocumentManagerController', () => {
 
       await documentManagerController.get(req, res);
 
-      expect(mockGet).toHaveBeenCalledWith({
-        url: 'https://ccd-case-document-am-api-prl-ccd-definitions-pr-541.service.core-compute-preview.internal/cases/documents/95f7c1be-f880-49db-b192-6632f43742b4/binary',
-      });
+      // expect(mockGet).toHaveBeenCalledWith({
+      //   url: 'https://ccd-case-document-am-api-prl-ccd-definitions-pr-541.service.core-compute-preview.internal/cases/documents/95f7c1be-f880-49db-b192-6632f43742b4/binary',
+      // });
```
Comment (AlokDatta) [inline src/main/app/document/DocumentManagementController.test.ts:87]: remove them if not required

File: src/main/steps/prl-cases/dashboard/controller/GetCaseController.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  public async getCase(req: AppRequest, res: Response): Promise<void> {
+    if (req.params?.caseId && req.session?.userCaseList) {
+      req.session?.userCaseList.forEach((element: CaseWithId) => {
+        if (element?.id.toString() === req.params?.caseId) {
+          req.session.userCase = element;
+        }
+      });
+    }
+    if (req.params?.caseId) {
+      const caseworkerUser = await getSystemUser();
```
Comment (AlokDatta) [inline src/main/steps/prl-cases/dashboard/controller/GetCaseController.ts:24]: change the constant reference

**Review by AlokDatta — COMMENTED** [review]
Minor comments

**Comment by sahaSonali** [comment]
Closing it as this code is already merged to feature/common-code-refactor branch

**Pull request closed without merging**